src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt

```diff
@@ -78,7 +78,7 @@ class ZaakafhandelParametersRESTService @Inject constructor(
     @GET
     @Path("caseDefinition")
     fun listCaseDefinitions(): List<RESTCaseDefinition> {
-        PolicyService.assertPolicy(policyService.readOverigeRechten().beheren)
+        assertPolicy(policyService.readOverigeRechten().beheren)
```

Comment:
Why not lift these assertions up into an annotation?

Reply:
Not a bad idea for the general use-case. @bas-info-nl would you create a story for that?

I think we misuse assertPolicy in a few places and throw all kinds of checks in, so we may still need the function.

Reply:
That would be nice yes but I think there may be some downsides to using annotations here like?
- will it not make unit testing harder?
- some policy checks need code to run first before they can be checked. These policies are not on the first line in the function/method in question